x-pack/plugins/actions/server/feature.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ACTION_SAVED_OBJECT_TYPE, ACTION_TASK_PARAMS_SAVED_OBJECT_TYPE } from './saved_objects';
+
+export const ACTIONS_FEATURE = {
+  id: 'actions',
+  name: i18n.translate('xpack.actions.featureRegistry.actionsFeatureName', {
+    defaultMessage: 'Actions',
+  }),
+  icon: 'bell',
+  navLinkId: 'actions',
+  app: [],
```

Comment:
It does using the `show` ui capability.
This is temporary as. once this PR is merged, this will use the proper management section in the feature here: https://github.com/elastic/kibana/pull/72029